Content.Client/Administration/UI/Logs/AdminLogsControl.xaml.cs

```diff
@@ -235,10 +244,25 @@ private bool ShouldShowPlayer(AdminLogPlayerButton button)
 
     private bool ShouldShowLog(AdminLogLabel label)
     {
-        return SelectedTypes.Contains(label.Log.Type) &&
-               (SelectedPlayers.Count + label.Log.Players.Length == 0 || SelectedPlayers.Overlaps(label.Log.Players)) &&
-               SelectedImpacts.Contains(label.Log.Impact) &&
-               label.Log.Message.Contains(LogSearch.Text, StringComparison.OrdinalIgnoreCase);
+        // Check log type
+        if (!SelectedTypes.Contains(label.Log.Type))
+            return false;
+
+        // Check players
+        if (!(IncludeNonPlayerLogs && SelectedPlayers.Count + label.Log.Players.Length == 0 ||
```

Comment:
This should be a valid case even when not including playerless logs since you aren't filtering for players which is how it used to work too.

```suggestion
        if (!(SelectedPlayers.Count + label.Log.Players.Length == 0 ||
```